gene2phenotype_project/gene2phenotype_app/serializers.py

```diff
@@ -1,10 +1,13 @@
+import json
+from deepdiff import DeepDiff
+import copy
+from collections import OrderedDict
```

Comment:
Add the new packages to requirements.txt

Reply:
Updated